docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
+            <q-grid:td> 
+                <i title="{{$column.name}}">{{$column.label}}</i>
+            </q-grid:td>
+        </q-grid:head>
+        <q-grid:body>
+            <q-grid:tr>
+            </q-grid:tr>
+            <q-grid:td key="firstName">
+                <b>{{$value}}</b>
+            </q-grid:td>
+            <q-grid:td key="dateBirth">
```

Comment:
yes actually complicated, logic like in your case can be put inside of the template
but still maybe it will be usefull to have that kind of triggers
maybe we will support 2variants?
`<g-grid:td on:mode='edit'>` - doesn't require watch
`<g-grid:td on='$mode === edit'>` requires watch?